cumulusci/core/tests/test_utils.py

```diff
@@ -34,6 +36,43 @@ def test_process_list_arg(self):
         self.assertEqual(["a", "b"], utils.process_list_arg("a, b"))
         self.assertEqual(None, utils.process_list_arg(None))
 
+    def test_process_glob_list_arg(self):
+        with temporary_dir():
+            touch("foo.py")
+            touch("bar.robot")
+
+            # Expect passing arg as list works.
+            self.assertEqual(
+                ["foo.py", "bar.robot"],
+                utils.process_glob_list_arg(["foo.py", "bar.robot"]),
+            )
+
+            # Falsy arg should return an empty list
+            self.assertEqual([], utils.process_glob_list_arg(None))
+            self.assertEqual([], utils.process_glob_list_arg(""))
+            self.assertEqual([], utils.process_glob_list_arg([]))
+
+            # Expect output to be in order given
+            self.assertEqual(
+                ["foo.py", "bar.robot"],
+                utils.process_glob_list_arg("foo.py, bar.robot"),
+            )
+
+            # Expect sorted output of glob results
+            self.assertEqual(["bar.robot", "foo.py"], utils.process_glob_list_arg("*"))
+
+            # Patterns that don't match any files
+            self.assertEqual(
+                ["*.bar", "x.y.z"], utils.process_glob_list_arg("*.bar, x.y.z")
+            )
+
+            # Recursive
+            os.mkdir("subdir")
+            touch("subdir/baz.resource")
+            self.assertEqual(
+                ["subdir/baz.resource"], utils.process_glob_list_arg("**/*.resource")
+            )
```

Comment:
This test is failing in Windows due to the different path separator. It looks like the glob does the right thing but you need to use os.path.join to build the expected result.